Pipfile

```diff
@@ -0,0 +1,12 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+invoke = "*"
+
+[dev-packages]
+
+[requires]
+python_version = "3.6"
```

Comment:
We must not tell other people what specific Python version they need to use

Reply:
@karihovi-vg You said you figured it out. Push your changes

Reply:
Yes, the fix for the python version requirement was simply to delete that python line under [requires]

Reply:
Well it doesn't look like the change is still here .. so what are you going to do about it?